browser/js/adminDash/song/adminDash.songs.html

```diff
@@ -1,7 +1,7 @@
 <ul>
   <span ng-repeat="song in songs">
-    <h3>{{song.title}}</h3>
-    <button ui-sref="songForm">Edit</button>
+    <h3><a ng-href="/songs/{{song.id}}">{{song.title}}</a></h3>
```

Comment:
Is there a reason to use an ng-href instead of ui-sref=songs({songId: song.id})?  I assume they do the same thing - is one better than the other?  We have been using ui-sref.